pkg/daemon/update.go

```diff
@@ -114,9 +114,6 @@ func reloadService(name string) error {
 }
 
 // performPostConfigChangeAction takes action based on what postConfigChangeAction has been asked.
-// For non-reboot action, it applies configuration, updates node's config and state.
-// In the end uncordon node to schedule workload.
-// If at any point an error occurs, we reboot the node so that node has correct configuration.
```

Comment:
Are these comments out of date? Or am I misunderstanding how error handling works here? It looks like it just returns errors